### PR TITLE
Add troubleshooting doc

### PR DESCRIPTION
**Pull request by jesmrec**
Implements #2554

Adding the troubleshooting doc, as in the other clients.

Document added to the root folder, since te `doc` and `docs` folders aims to the automatic documentation processes handled by antora.

Let me know any issue or improvement

### CODE REVIEW

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+problems and, if you report a bug, log output can help to resolve an issue more
+quickly.
+
+Here you have a description about how to get relevant information to be handled in case of any issue. You can ship this information via mail (android-app@owncloud.com), creating a new issue in the [open GitHub ownCloud Android repository](https://github.com/owncloud/android), or in [public ownCloud forum](https://central.owncloud.org/).
+
+### Capturing App Debug Logs
+
+You will find the Logs in the Settings view of the Android app. If the Logs option is not visible, tap five times on the build number (bottom of the Settings view), and Logs will be an option inside the "More" section.
+
+When you have reproduce the issue you want to address, the "Send History" button will automatically add the logs to a new mail message.
```
Comment (davigonz) [inline TROUBLESHOOTING.md]: When you have reproduce**d** the issue

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+1. Delete History
```
Comment (davigonz) [inline TROUBLESHOOTING.md]: Maybe a description before explaining what are these steps for? Something like:

_When you have reproduced the issue you want to address, the "Send History" button will automatically add the logs to a new mail message. Please follow the next steps to send us your logs_

**Comment by davigonz** [comment]
@jesmrec some changes requested here...

File: TROUBLESHOOTING.md (hunk 1 above)
Comment (jesmrec) [reply TROUBLESHOOTING.md]: fixed

File: TROUBLESHOOTING.md (hunk 2 above)
Comment (jesmrec) [reply TROUBLESHOOTING.md]: good idea! added.

**Comment by jesmrec** [comment]
Changes done

**Comment by davigonz** [comment]
@jesmrec ready to go, I guess no QA is needed here

**Comment by davigonz** [comment]
@jesmrec can you rebase so we can merge it? Thanks

**Comment by jesmrec** [comment]
no, this is ready to go.

**Pull request merged**